src/test/kotlin/lotto/FixedLottoNumberGenerator.kt

```diff
@@ -0,0 +1,10 @@
+package lotto
+
+import lotto.domain.LottoNumber
+import lotto.util.LottoNumberGenerator
+
+class FixedLottoNumberGenerator : LottoNumberGenerator {
+    override fun generate(): Set<LottoNumber> {
+        return (1..6).map { LottoNumber.getNumber(it) }.toSet()
```

Comment:
생성자로 원하는 로또 번호를 주입받는 구조면
원하는 로또넘버를 만들수 있을거같네요 :)

Reply:
해당 메서드는 LottoNumberGenerator의 메서드를 오버라이드한 메서드라서 값을 넣도록 바꾸게 되면 RandomGenerator에서는 불필요한 파라미터를 받게될거 같습니다!
필요하다면 FixedNumberGenerator 클래스 내에 별도의 메서드를 만드는 방향이 나을거 같은데 어떻게 생각하실까요?

Reply:
```suggestion
class FixedLottoNumberGenerator(private val lottoNumbers:List<Int>) : LottoNumberGenerator {
    override fun generate(): Set<LottoNumber> {
        return lottoNumbers.map { LottoNumber.getNumber(it) }.toSet()
```
와 같이 추상화된 인터페이스와는 별개로 구현체를 수정할수 있을거같아요 :)

Reply:
이 방식은 생각을 하지 못했었네요! 감사합니다:)